src/custom/components/Menu/index.tsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react'
-import { Code, MessageCircle } from 'react-feather'
+import { Code, MessageCircle, IconProps } from 'react-feather'
 
-import MenuMod, { MenuItem } from './MenuMod'
+import MenuMod, { MenuItem, InternalMenuItem } from './MenuMod'
 import styled from 'styled-components'
+import { FOOTER_URLS } from '../Footer'
 
 const CODE_LINK = 'https://github.com/gnosis/dex-swap'
 const DISCORD_LINK = 'https://chat.gnosis.io'
 
 export const StyledMenu = styled(MenuMod)``
 
+// remove static gnosis text and leave urls
+const CONTENT_URLS = FOOTER_URLS.slice(1) as { name: string; url: string; Icon?: React.FC<IconProps> }[]
```

Comment:
Why not remove it from the `FOOTER_URLS` then?

Reply:
yep, i think we are removing the footer links. They will dissapear when we consolidate things.